src/annotator/components/AdderToolbar.js

```diff
@@ -34,55 +55,60 @@ function NumberIcon({ badgeCount }) {
  */
 function AdderToolbarArrow({ arrowDirection }) {
   return (
-    <Icon
-      name="pointer"
-      classes={classnames(
-        // Position the arrow in the horizontal center at the bottom of the
-        // container (toolbar). Note: the arrow is pointing up at this point.
-        'absolute left-1/2 -translate-x-1/2',
-        // Override `1em` width/height rules in `Icon` to size the arrow as
-        // its SVG dimensions dictate
-        'h-auto w-auto z-2',
-        'text-grey-3 fill-white',
+    <div
+      className={classnames(
+        // Position horizontally center of the AdderToolbar
+        'absolute left-1/2 -translate-x-1/2 z-2',
+        'fill-white text-grey-3',
         {
-          // Down arrow: transform to point the arrow down
-          'rotate-180': arrowDirection === 'down',
-          // Up arrow: position vertically above the toolbar
+          // Move the pointer to the top of the AdderToolbar
           'top-0 -translate-y-full': arrowDirection === 'up',
         }
       )}
-    />
+    >
+      {arrowDirection === 'up' ? <PointerUpIcon /> : <PointerDownIcon />}
+    </div>
   );
 }
 
 /**
  * @param {object} props
  *  @param {number} [props.badgeCount]
- *  @param {string} [props.icon]
+ *  @param {IconComponent} [props.Icon]
  *  @param {string} props.label
  *  @param {() => void} props.onClick
  *  @param {string|null} props.shortcut
  */
-function ToolbarButton({ badgeCount, icon, label, onClick, shortcut }) {
+function ToolbarButton({ badgeCount, Icon, label, onClick, shortcut }) {
   useShortcut(shortcut, onClick);
 
   const title = shortcut ? `${label} (${shortcut})` : label;
 
   return (
-    <LabeledButton
-      className={classnames(
-        'flex flex-col gap-y-1 items-center py-2.5 px-2',
-        'text-annotator-sm leading-none text-grey-7',
-        'transition-colors duration-200',
-        'dim-item'
+    <ButtonBase
+      classes={classnames(
+        'flex-col gap-y-1 py-2.5 px-2',
+        'text-annotator-sm leading-none',
+        // Default color when the toolbar is not hovered
+        'text-grey-7',
+        // When the parent .group element is hovered (but this element itself is
+        // not), dim this button's text. This has the effect of dimming inactive
+        // buttons.
+        'group-hover:text-grey-5',
+        // When the parent .group element is hovered AND this element is
+        // hovered, this is the "active" button. Intensify the text color, which
+        // will also darken any descendant Icon
+        'hover:group-hover:text-grey-9',
+        // Set the .hover-group class for use by `NumberIcon`
+        'hover-group'
```

Comment:
This use case is a good example of both the pros and cons of `ButtonBase`. On the "pros" side, we automatically get consistent transitions and focus rings and whatnot, so those don't have to be managed manually, nor risk being inconsistent. On the other hand, I just kind of have to "know" that `ButtonBase` already sets a `flex` layout, so I just need to set `flex-col` to go columnar. But I can feel assured that padding and colors won't conflict, as the coalescing convention for base classes is never to style dimensions or colors...